jerry-core/debugger/jerry-debugger.h

```diff
@@ -82,12 +107,17 @@ typedef enum
   JERRY_DEBUGGER_EVAL_ERROR_END = 20, /**< last part of eval result when an error is occured */
 
   /* Messages sent by the client to server. */
+    /* The following messages are accepted in both run and breakpoint modes. */
```

Comment:
I don't have strong preferences. Those are only comments. :) Their content is the important part. I've already approved the PR.